<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>4000</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Staircase Simulator v 0.5</string>
  </property>
  <property name="iconSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>580</width>
      <height>464</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSimulator">
     <attribute name="title">
      <string>Simulator</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Simulator parameters</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="toolTip">
                  <string>Set the number of repetition of a full staircase procedure to simulate.</string>
                 </property>
                 <property name="text">
                  <string>Simulator repetitions</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxSimRepetitions">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Num of staircases</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBoxNStaircases">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Psychometric function</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>Normal cumulative </string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Logistic (not implemented yet)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cauchy (atan)  (not implemented yet)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Right gumbel  (not implemented yet)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Left gumbel  (not implemented yet)</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Mean</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxMean">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Standard deviation</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxStd">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>0.001000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>GuessRate</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxGuessRate">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>LapseRate</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxLapseRate">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="buttonStart">
             <property name="text">
              <string>Start!</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetButton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="TabStaircase" name="tabStaircase"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tabAnalysis">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Statistics</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QFormLayout" name="formLayout_4">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBoxSortByID">
                  <property name="text">
                   <string>Sorted by ID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Num of trials required</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="labelNTrialsRequired">
                  <property name="text">
                   <string>--</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Reversals to average</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBoxLastInvAverage">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>999999</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Average on last reversals</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listWidget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="isWrapping" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Psychometric targeting</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listWidgetPsychoTarget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="isWrapping" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Data</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tabGraph">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="title">
      <string>Graph</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Line style:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDrawStyle">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cross</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circles</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Squares</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Show reference psychometric</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="customPlot" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBayesian_Model">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_experiment"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionExportResultsToFile"/>
    <addaction name="actionExport_graph_to_pdf"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>Credits</string>
    </property>
    <addaction name="actionCarlo_Nicolini_2012_carlo_nicolini_iit_it"/>
   </widget>
   <addaction name="menuBayesian_Model"/>
   <addaction name="menuCredits"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open experiment file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save experiment file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExportResultsToFile">
   <property name="text">
    <string>Export results to file...</string>
   </property>
   <property name="toolTip">
    <string>Save the tabular content to a csv file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNew_experiment">
   <property name="text">
    <string>New experiment...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExport_graph_to_pdf">
   <property name="text">
    <string>Print graph to pdf...</string>
   </property>
   <property name="toolTip">
    <string>Export the graph with staircases to a pdf file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCarlo_Nicolini_2012_carlo_nicolini_iit_it">
   <property name="text">
    <string>Carlo Nicolini 2012 - carlo.nicolini@iit.it</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabStaircase</class>
   <extends>QTabWidget</extends>
   <header>TabStaircase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
